aws-flow/lib/aws/decider/generic_client.rb

```diff
@@ -85,7 +85,7 @@ def bail_if_external
       #      foo.send :bar # plus args and block if appropriate
       #    end
       #
-      def send_async(task, *args, &block)
+      def send_async(task, **args, &block)
```

Comment:
This looks like it is taking away the functionality for positional arguments for keyword args.

Reply:
If one splat is used it splits the positional arguments from an array into individual arguments.
But in ruby 3 keyword arguments are treated separately from positional arguments. so *args would not contain keyword arguments that are passed to it. You change allows it to accept keyword arguments but not positional arguments.

Reply:
I played around with it in pry
```
def async(task, *args, **kwargs)
  send(task, *args, **kwargs)
end
[29] pry(main)> def met3(a, b:)
  p a+b
=> :met3
[33] pry(main)> async(:met3, 2, b: 3)
5
=> 5
```
As you can see I am able to accept and pass both positional and keyword arguments. This worked on both ruby 2.7 and 3.1.2